frontend/packages/component-library/src/tooltip/WithTooltip.tsx

```diff
@@ -103,19 +103,15 @@ const WithTooltip: React.FunctionComponent<WithTooltipProps> = ({
   // Effect to handle the Escape key to close the tooltip
   useEffect(() => {
     const handleKeyDown = (event: KeyboardEvent) => {
-      if (event.key === 'Escape' && !event.defaultPrevented) {
-        if (event.target instanceof HTMLElement) {
-          event.target.blur(); // Remove focus from the container
```

Comment:
Q about `blur` in general - is this what we'll need to use to address https://codedotorg.atlassian.net/browse/CT-1164?

Reply:
Blur could possibly work! I'm not entirely sure how it's getting focus again so it might be a matter of deleting code.. we'll have to get in there and find out😃

Reply:
Thanks!